### PR TITLE
Several updates:

### PR DESCRIPTION
**Pull request by yakutovicha**
* bump voila-aiidalab-template to 0.2.1
* bump aiidalab-optimade to v1.2.1
* downgrade ase to 3.19.2 to fix https://github.com/lsmo-epfl/aiida-lsmo/issues/46

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -207,16 +206,16 @@ sqlalchemy-utils==0.34.2
...
 terminado==0.9.1; python_version >= '3.6'
 testpath==0.4.4
 threadpoolctl==2.1.0; python_version >= '3.5'
-toml==0.10.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'
-topika==0.2.1
+toml==0.10.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
+topika==0.2.2
 tornado==4.5.3
-tqdm==4.51.0; python_version >= '2.6' and python_version not in '3.0, 3.1'
+tqdm==4.51.0; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
```
Comment (csadorf) [inline requirements.txt:218]: Do you know why pipenv constantly changes its mind on the excluded Python versions?

**Review by csadorf — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (yakutovicha) [reply requirements.txt:218]: I've absolutely no idea... The Pipenv version I am using is the same.

Comment (csadorf) [reply requirements.txt:218]: Odd.

**Review by csadorf — DISMISSED** [review]

File: Pipfile [hunk 2]
```diff
@@ -28,6 +28,7 @@ aiida-vasp = "~=0.2.4"
 aiida-wannier90 = "~=2.0"
 aiida-yambo = "~=1.0"
 aiida-zeopp = "~=1.1"
+ase = "<3.20"
```
Comment (csadorf) [inline Pipfile:31]: Sorry, I just realized that we should probably add a comment here as to the reasoning for this constraint.

**Review by csadorf — APPROVED** [review]

File: Pipfile (hunk 2 above)
Comment (yakutovicha) [reply Pipfile:31]: added.

**Review by csadorf — DISMISSED** [review]

**Pull request merged**